### PR TITLE
:ear_of_rice: Make the Rook Ceph as default storageclass

### PR DESCRIPTION
**Pull request by mostafaghadimi**
**Description of your changes:**
This file will resolve the issue that in some cases there isn't any default storageclass.
**Which issue is resolved by this Pull Request:**
Resolves #

**Checklist:**

- [x] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/latest/Contributing/development-flow/#commit-structure)).
- [ ] **Skip Tests for Docs**: If this is only a documentation change, add the label `skip-ci` on the PR.
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/latest/Contributing/development-flow/#submitting-a-pull-request)
- [ ] [Pending release notes](https://github.com/rook/rook/blob/master/PendingReleaseNotes.md) updated with breaking and/or notable changes for the next minor release.
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.

### CODE REVIEW

**Comment by ideepika** [comment]
@mostafaghadimi I was investigating github actions failures, found this one looks related to the PR, in case you are not already aware, adding here:
https://github.com/rook/rook/runs/8086813786?check_suite_focus=true

File: deploy/examples/csi/rbd/storageclass.yaml [hunk 1]
```diff
@@ -18,6 +18,9 @@ apiVersion: storage.k8s.io/v1
 kind: StorageClass
 metadata:
   name: rook-ceph-block
+  # annotation:
+  #   storageclass.kubernetes.io/is-default-class: true
+    storageclass.kubernetes.io/is-default-class: true
```
Comment (travisn) [inline deploy/examples/csi/rbd/storageclass.yaml:23]: Please [squash the commits](https://rook.io/docs/rook/latest/Contributing/development-flow/#commit-history) and also remove this line.

**Review by travisn — CHANGES_REQUESTED** [review]

File: deploy/examples/csi/rbd/storageclass.yaml (hunk 1 above)
Comment (mostafaghadimi) [reply deploy/examples/csi/rbd/storageclass.yaml]: Dear @travisn,

Thanks for your helpful comment. 

Sincerely,
Mostafa

**Comment by mostafaghadimi** [comment]
@travisn I’m done.

**Comment by travisn** [comment]
@mostafaghadimi There was a problem with the rebase, see there are five commits instead of one. Please [rebase](https://rook.io/docs/rook/latest/Contributing/development-flow/#commit-history) again so there is a single commit, thanks

**Pull request closed without merging**